x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```diff
@@ -118,3 +118,26 @@ query = "file where 66.0 / serial_event_id == 1"
 [[queries]]
 expected_event_ids = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 46]
 query = "process where serial_event_id + ((1 + 3) * 2 / (3 - 1)) * 2 == 54 or 70 + serial_event_id < 100"
+
+[[queries]]
+query = '''
+sequence
```

Comment:
Apparently these are getting dropped  by CommonEqlActionTestCase.readTestSpecs.
Any idea why @aleksmaus?

Reply:
Ah, they still need to be  removed from `test_queries_unsupported.toml`

Reply:
And since these are duplicates of test_queries.toml, you can remove them here too